### DISCUSSION TITLE
Zoom in/out using command key instead of control key (MacOS)

### DISCUSSION THREAD

**Discussion by knowine** [freeplane/freeplane#1347] [Help]
Is there a way to change zoom in/out shortcut from **control** + mouse wheel scroll to **command** + mouse wheel scroll? Since I'm on mac, this would be really convenient

**Comment by Illegit** [comment]
Hi, I use 2 command keys 1 for each of these scripts, zoom in 5% and zoom out 5%.
If they can be useful. I use them in w11.
[zoom5.zip](https://github.com/freeplane/freeplane/files/12369430/zoom5.zip)

**Comment by Illegit** [comment]
You have to add those scripts in the folder named "scripts" in your FP user folder: 
eg: C:\Users\YOUR_USER_NAME\AppData\Roaming\Freeplane\1.11.x\scripts
